js/openapi-ts.config.js

```diff
@@ -0,0 +1,17 @@
+//@ts-ignore
+import { defineConfig } from "@hey-api/openapi-ts";
+
+export default defineConfig({
```

Comment:
what does this file do and shouldn't it be pointed to staging/prod (with additional support for `process.env.BACKEND_URL`)